contracts/DaiHrd.sol

```diff
@@ -468,6 +468,11 @@ contract MakerFunctions {
 
 
 contract DaiHrd is ERC777, MakerFunctions {
+	event Deposit(address indexed from, uint256 attodai);
```

Comment:
May be helpful to include the amount of DaiHrd that was issued.  Technically that can be derived off chain, but it will be a huge pain.

Reply:
Same for withdrawal, would be helpful to include DaiHrd amount.